### PR TITLE
1.9 update integration script

### PR DESCRIPTION
**Pull request by renato-bylyk**
This PR updates the integration script for magento 1.9 to match the lastest (25th February, 2019) e-satisfaction integration guidelines.

### CODE REVIEW

File: app/design/frontend/base/default/template/esatisfaction/header.phtml [hunk 1]
```diff
@@ -5,9 +5,24 @@ if ($status) {
...
             };
+
+            // Setup script
+            doc.addEventListener('DOMContentLoaded', function () {
+                var body = doc.getElementsByTagName('body')[0], script = doc.createElement('script');
+                script.async = true;
+                script.src = 'https://collection.e-satisfaction.com/dist/js/integration' + (!!jq ? '' : '.jq') + '.min.js';
+                body.appendChild(script);
+            });
+        })(window, document, "<?php echo $applicationId; ?>", false, {});
```
Comment (ioannis-papikas) [inline app/design/frontend/base/default/template/esatisfaction/header.phtml]: There is an option (on deleted footer code) about jquery enabled. Use it here to define whether we need jQuery version or not.

**Review by ioannis-papikas — CHANGES_REQUESTED** [review]

File: app/design/frontend/base/default/template/esatisfaction/header.phtml [hunk 2]
```diff
@@ -5,9 +5,24 @@ if ($status) {
...
             };
+
+            // Setup script
+            doc.addEventListener('DOMContentLoaded', function () {
+                var body = doc.getElementsByTagName('body')[0], script = doc.createElement('script');
+                script.async = true;
+                script.src = 'https://collection.e-satisfaction.com/dist/js/integration' + (!!jq ? '' : '.jq') + '.min.js';
+                body.appendChild(script);
+            });
+        })(window, document, "<?php echo $applicationId; ?>", "<?php $helperData->getIsJQueryEnabled() ? true : false ?>", {});
```
Comment (ioannis-papikas) [inline app/design/frontend/base/default/template/esatisfaction/header.phtml]: I think it should be the other way around with the `true` `false` parameters.

Comment (renato-bylyk) [reply app/design/frontend/base/default/template/esatisfaction/header.phtml]: The previous check was:
`$jqueryInstalled = $helperData->getIsJQueryEnabled();`	
`$fileName = $jqueryInstalled ? 'integration.jq.min.js' : 'integration.min.js';`

Comment (ioannis-papikas) [reply app/design/frontend/base/default/template/esatisfaction/header.phtml]: We have to check the plugin's admin panel to see what's the use of the variable.
The same should apply to the 2.2 version as well.

Comment (ioannis-papikas) [reply app/design/frontend/base/default/template/esatisfaction/header.phtml]: From magento admin panel, helper text about this variable is:

> Select yes if jquery is installed in your magento theme installation

So, let's change the code to behave accordingly.

**Comment by renato-bylyk** [comment]
Will reopen with new changes.

**Pull request closed without merging**